docs/dictionary/control_st/if.lcdoc

```diff
@@ -40,35 +40,32 @@ statement.
 
 Description:
 Use the <if> <control structure> to <execute> a <statementList|statement> 
-(or list of <statementList|statements>) only under certain circumstances. Form:The <if>
-<control structure> always begins with the <word> if. There are four
-forms of the <if> <control structure>:
-
-```
-if condition then statementList [else elseStatementList]
-```
-(This form may have a line break before the words then or else or both.)
-
-```
-if condition then
-   statementList 
-[else
-   elseStatementList]
-end if
-```
-```
-if condition 
-then statement 
-[else
-   elseStatementList]
-end if
-```
-```
-if condition then
-   statementList 
-else elseStatement
-end if
-```
+(or list of <statementList|statements>) only under certain circumstances. 
+#####Form: 
```

Comment:
@seaniepie I think this can just be removed. At some point we will probably change lcdoc to use `#Element` headings instead of `Element:` so it is probably better not to use `#` at all until we do that. I don't really thing `Form:` adds anything to the description here.